### DISCUSSION TITLE
What about a collab with @vladmandic ?

### DISCUSSION THREAD

**Discussion by Nacurutu** [anapnoe/stable-diffusion-webui-ux#54] [Ideas]
Hey @anapnoe hope u r doing great, again, great job with ur UI/UX design.

I was talking with @vladmandic and mentioned to him that ur UI/UX is really great... so, maybe you can work together and create a fork merging ur changes and his changes. He also has a fork of the webUI with a lot of fixes and features and is getting more visibility right now: 

https://github.com/vladmandic/automatic

He mentioned that if you are interested, you both can work together and merge everything from both forks, you can read his response here:

https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/9505#discussioncomment-5596286

What do you think? Let us know...

**Comment by halr9000** [comment]
IMHO these two repos seem to be the best and only ones ready to succeed auto's. A collab would not be a bad thing at all.

**Reply by vladmandic** [reply]
we have opened communication and trying to figure out how to proceed.

**Reply by halr9000** [reply]
> we have opened communication and trying to figure out how to proceed.

Nice.

**Reply by anapnoe** [reply]
this is going to be a great journey!

**Reply by ScrudgMcDack** [reply]
Guys, this is amazing news! I'm thrilled! And anticipating a truly amazing result!
Can you please share with us the current state of affairs and at least a hint of an anticipated release date?  :)

**Reply by vladmandic** [reply]
@ScrudgMcDack Nope :)
Just kidding - we'll share when we have something more concrete. At the moment, things are very fluid and hectic.

**Reply by myndxero** [reply]
> this is going to be a great journey!

Random question from Vlad's FAVORITE guy with questions, not sure if your avatar has anything to do with penrose triangles, but out of random curiosity, was wondering if you've ever gotten SD to produce a proper penrose triangle? That is all, can'twait for the fruits of this collab!

**Comment by 779575267** [comment]
After using it again today, I feel that the compatibility and usability have improved compared to vladmandic's.
